xml/System.Reflection.Metadata.Ecma335/MetadataBuilder.xml

```diff
@@ -1578,10 +1758,10 @@
         <Parameter Name="allowUnpairedSurrogates" Type="System.Boolean" Index="1" FrameworkAlternate="netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="allowUnpairedSurrogates">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The constant value to add.</param>
```

Comment:
Not sure, but probably not. To avoid making this param description restricted to something that it shouldn't, I'll remove the `constant` word:

```suggestion
        <param name="value">The value to add.</param>
```